runtime_config_linux.go

```diff
@@ -198,6 +198,12 @@ type Network struct {
 	Priorities []InterfacePriority `json:"priorities"`
 }
 
+// IntelRdt for Linux cgroup 'intel_rdt' resource management
+type IntelRdt struct {
+	// L3 cache capacity bitmask (CBM) for container
+	L3Cbm *string `json:"l3Cbm,omitempty"`
```

Comment:
On Tue, Dec 22, 2015 at 09:22:58PM -0800, Xiaochen Shen wrote:
> Case (2), '4080' is very confusing as a bitmask value for end user.

The counter-argument here is that the target audience for the
config-file isn't end users [1,2,3], which can always insert some
tooling in between themselves and the JSON config.  For example, it
wouldn't be too bad to write a parser that converts a JSON-extension
form like:

  "intel_rdt": {
    "l3Cbm": 0xff0
  }

to official JSON.  And it's even easier (from an implementation
perspective, if not from a usability perspective) to do:

  $ python -c 'print(hex(4080))'
  0xff0

So my preference is “use integer types for integer values, and deal
with the lack of JSON hex support using other tools”.  But I don't
feel strongly enough about that put up a fight if the consensus is
that we want to use JSON strings for integers that are more intuitive
as hex.

[1]: https://www.opencontainers.org/news/faqs/how-broad-mission-oci
[2]: https://groups.google.com/a/opencontainers.org/d/msg/dev/qWHoKs8Fsrk/duzW7SIGDQAJ
     Subject: Re: removal of cgroups from the OCI Linux spec
     Date: Thu, 29 Oct 2015 20:15:13 -0700
     Message-ID: <CACfVidd8sV14rG6S2DX1mJ35ij1-Y7sAxir4p-U9EscCeqCMwA@mail.gmail.com>
[3]: https://github.com/opencontainers/specs/pull/275#discussion_r48074805